src/app/calendar/layout.tsx

```diff
@@ -0,0 +1,13 @@
+import type { ReactNode } from 'react';
+
+const RootLayout = ({ children }: { children: ReactNode }) => {
```

Comment:
형식 통일해주시고, 이 레이아웃이 Root는 아니니 네이밍 맞게 변경하면 좋을 것 같아요!
```suggestion
interface LayoutProps{
  children: ReactNodel
}
const Layout = ({ children }: LayoutProps) => {
```